### DISCUSSION TITLE
Give me the old feed.

### DISCUSSION THREAD

**Discussion by DianaNites** [community/community#68111] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

You want feedback, add me to the pile. You keep popping up such a big banner asking for it instead of just showing me the information i want exclusively from the people i choose to follow, so.

**Comment by HoutarouOreki** [comment]
Yep. 

I want to only see new things to discover, but instead it's filled with repositories and people I follow.

**Comment by pprp** [comment]
Yes. I can't see anything that attracts me anymore! We wanna the old feed!
